stream-chat-android-client/src/main/java/io/getstream/chat/android/client/helpers/AttachmentHelper.kt

```diff
@@ -15,7 +17,12 @@ public class AttachmentHelper(private val systemTimeProvider: SystemTimeProvider
         return timestamp > systemTimeProvider.provideCurrentTimeInSeconds()
     }
 
+    public fun hasStreamImageUrl(attachment: Attachment): Boolean {
+        return attachment.imageUrl?.toHttpUrlOrNull()?.host?.contains(STREAM_CDN_HOST_PART, ignoreCase = true) ?: false
+    }
+
     private companion object {
         private const val QUERY_KEY_NAME_EXPIRES = "Expires"
+        private const val STREAM_CDN_HOST_PART = "stream-chat"
```

Comment:
Couldn't it gives us some issues in the future if another customer is using a CDN with the following url:
`stream-chat-cdn.MyCompany.com` ??

Reply:
You're right. Changed it to more specific regex